prow/plugins/approve/approvers/README.md

```diff
@@ -23,6 +23,7 @@ Reviewers:
 
 An example of the OWNERS file is listed below:
 
+```yaml
 reviewers:
 * jack
```

Comment:
Is this valid yaml?

Reply:
Silly me, I somehow thought that the `*` was also accepted as a list item. Fixed.